connector/auth/emulator_validation.go

```diff
@@ -42,5 +43,6 @@ func IsTokenFromEmulator(authHeader string) bool {
 	return true
 }
 
+// EmulatorTokenValidator provides functionality to check if a request is from an emulator
 type EmulatorTokenValidator struct {
 }
```

Comment:
We should probably remove this file if we are not using it

Reply:
Delete the file